coderd/database/dump.sql

```diff
@@ -1528,6 +1524,12 @@ CREATE VIEW template_with_names AS
 
 COMMENT ON VIEW template_with_names IS 'Joins in the display name information such as username, avatar, and organization name.';
 
+CREATE TABLE user_configs (
+    user_id uuid NOT NULL,
+    key character varying(256) NOT NULL,
+    value text NOT NULL
```

Comment:
Since we limit the key length, should we also limit the value length?

Reply:
I tend to prefer `text` for columns like this. a key isn't likely to be very long, but the value could theoretically be a long json blob, and you'll get bad results and a hard to diagnose bug if it gets chopped.